docs/Learning-Environment-Design-Agents.md

```diff
@@ -67,7 +67,9 @@ includes implementing the following methods:
 - `Agent.Heuristic()` - When the `Behavior Type` is set to `Heuristic Only` in
   the Behavior Parameters of the Agent, the Agent will use the `Heuristic()`
   method to generate the actions of the Agent. As such, the `Heuristic()` method
-  writes to a provided array of floats.
+  writes to the array of floats provided to the Heuristic method as argument.
```

Comment:
Would this be better on the Agent.Heuristic xmldoc instead?

Reply:
I added the _Note_ to the xmldoc. Let me know what you think